Languages/JS/questions.json

```diff
@@ -108,6 +108,51 @@
 		"trueOrFalse": false,
 		"shortAns": "It is an object, which object depends upon where the code is being invoked.",
 		"tag": "JavaScript"
+	},
+	{
+		"id": 12,
```

Comment:
This question and shortAns are way too obvious. Please phrase the shortAns as "It is ..." and please dont use the wording of the question itself in the answer, that is a dead give away.

please don't use the word in the shortAns.

"what does a cat eat?"
a) oranges
b) tacos
c) a cat eats mice
d) trees.

making the answer so obvious defeats the point of the app.